authentication.go

```diff
@@ -105,5 +106,14 @@ func (a *App) ValidateAuthToken(authToken string) (string, bool) {
 		return "", false
 	}
 
+	audience, ok := claims["aud"].([]string)
```

Comment:
the `aud` value may be a `[]string` or `string`

Reply:
Good call out! Will keep this in mind with the other languages as well.